docs/compute-element/htcondor-ce-overview.md

```diff
@@ -32,12 +32,22 @@ How Jobs Run
 
 Once an incoming grid job is authorized, it is placed into HTCondor-CE’s scheduler where the JobRouter creates a transformed copy (called the *routed job*) and submits the copy to the batch system (called the *batch system job*). After submission, HTCondor-CE monitors the batch system job and communicates its status to the original grid job, which in turn notifies the original submitter (e.g., job factory) of any updates. When the job completes, files are transferred along the same chain: from the batch system to the CE, then from the CE to the original submitter.
 
+### Over SSH
```

Comment:
I think I'd this section to lead with that it is the Hosted CE configuration (and avoid references to htcondor-ce-bosco), intended for small sites, contact support, etc. Then have the technical details after that.